packetery/src/Packetery/Checkout.php

```diff
@@ -440,4 +447,36 @@ private function getChosenMethod(): string {
 
 		return '';
 	}
+
+	/**
+	 * Gets carrier id from chosen shipping method.
+	 *
+	 * @param string $chosenMethod Chosen shipping method.
+	 *
+	 * @return string|null
+	 */
+	public function getCarrierId( string $chosenMethod ): ?string {
+		if ( strpos( $chosenMethod, self::CARRIER_PREFIX ) !== 0 ) {
+			return null;
+		}
+		$methodIdParts = explode( self::CARRIER_PREFIX, $chosenMethod );
+		if ( strpos( $methodIdParts[1], 'zpoint' ) === 0 ) {
+			// TODO: use constant Repository::INTERNAL_PICKUP_POINTS_ID.
```

Comment:
proč toto neuděláme hned ? Co nám v tom brání ?

Reply:
Protože už je to připraveno v https://github.com/Zasilkovna/WooCommerce/pull/38